src/Disposable.jsx

```diff
@@ -22,22 +22,34 @@ function disposeHelper(disposable) {
     }
 }
 
+const _disposables = Symbol('disposables');
+const _isDisposed = Symbol('isDisposed');
```

Comment:
Interesting naming convention with the leading underscore. It seemed strange to me initially, but I guess it makes it clearer that we're dealing with an internal property name, rather than an easily-confused variable with the same name.